docs/topics/i18n/translation.txt

```diff
@@ -2196,3 +2196,21 @@ to produce the binary ``.mo`` files that are used by ``gettext``.
 You can also run :djadmin:`django-admin compilemessages
 --settings=path.to.settings <compilemessages>` to make the compiler process all
 the directories in your :setting:`LOCALE_PATHS` setting.
+
+Non-English base language
+-------------------------
+
+Django makes the general assumption that the original strings in a translatable
+project are written in English.
+You can choose another language, but you must be aware of certain limitations:
```

Comment:
if this is meant as a separate paragraph it needs a blank line above; otherwise reflow to previous line